modelci/app/v1/endpoints/trainer.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 1/15/2021
+"""
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.post('/')
+def submit(model_id: str, other_body: ...):
+    """
+    Submit a training job.
+
+    Args:
+        model_id (str): Model ID for training.
```

Comment:
when do we generate the training_job_id?
I have two options.

1. Generate training_job_id here and return is as an output
2. Genetrate job_id in controller and input it to this function.


This should be designed very carefully. Please leave a todo here

Reply:
We use opt 1 (i.e. generate a job object with ID, save it into the database, submit it to the controller, and return it as an output)
Wil updated the workflow to README draft.